packages/fast-usdc/test/fast-usdc.contract.test.ts

```diff
@@ -56,10 +66,43 @@ test('start', async t => {
   const seat = await E(zoe).offer(inv);
   t.is(
     await E(seat).getOfferResult(),
-    'noop; evidence was pushed in the invitation maker call',
+    'noop; nothing should be expected from this offer',
   );
 });
 
+test('oracles', async t => {
+  const common = await commonSetup(t);
+  const { creatorFacet, zoe } = await startContract(common);
+
+  const oracleId = 'oracle-1';
+
+  const oracleInvitation = await E(creatorFacet).makeOracleInvitation(oracleId);
+  const oracleAdminKit = await E(
+    E(zoe).offer(oracleInvitation),
+  ).getOfferResult();
+
+  t.deepEqual(Object.keys(oracleAdminKit), [
+    'admin',
+    'invitationMakers',
+    'oracle',
+  ]);
+
+  const e1 = MockCctpTxEvidences.AGORIC_NO_PARAMS();
```

Comment:
I don't think the parameterization has been used yet, so we could remove this.

P.S. `AGORIC_NO_PARAMS` is meant to be an example of bad evidence, so this will start failing once the evidence validation TODO is completed.